waves/project.py

```diff
@@ -1539,7 +1746,7 @@ def technical_loss_ratio(self) -> float:
         elif self.turbine_type == "fixed":
             return 0.01
         else:
-            raise NotImplementedError("land-based is not currently modeled")
+            return 0.0
```

Comment:
I see here and for electrical losses that these are 0, is that coming from landbosse/COWER, or do we just not have a good source yet?